apps/src/applab/designMode.js

```diff
@@ -391,7 +411,9 @@ designMode.onDeletePropertiesButton = function(element, event) {
   if (isScreen) {
     designMode.loadDefaultScreen();
   } else {
-    designMode.editElementProperties(elementUtils.getPrefixedElementById(currentScreenId));
+    designMode.editElementProperties(
+        elementUtils.getPrefixedElementById(
+            Applab.reduxStore.getState().currentScreenId));
```

Comment:
I think we want to limit how much we do stuff like this (accessing the store directly). Ideally we wouldn't at all, but that might be difficult in a world where not all of our UI is react.